starter-code/functions-and-arrays.js

```diff
@@ -34,16 +42,25 @@ console.log(total);
 
 // Calculate the Average
 function averageNumbers (array) {
-
+  return sumArray(array) / array.length;
 }
 
 var numbers = [2, 6, 9, 10, 7, 4, 1, 9];
 var average = averageNumbers(numbers);
 console.log(average);
 
+// Calculating a Sum words
+function sumArrayWords (array) {
+  var sum = 0;
+  array.forEach(function(elmt){
```

Comment:
here you should have used `word` as the value in the function
it's ok to copy paste 😉 but make sure to adapt the content